presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisConfig.java

```diff
@@ -59,6 +60,19 @@ public KinesisConfig setTableDescriptionLocation(String tableDescriptionLocation
         return this;
     }
 
+    public Duration getUpdateInterval()
+    {
+        return updateInterval;
+    }
+
+    @Config("kinesis.update-interval")
+    @ConfigDescription("S3 configurable Kinesis update interval")
```

Comment:
How about `kinesis.table-descritpion-refresh-interval` and "How often to get the table description from S3"?